main.go

```diff
@@ -218,6 +220,25 @@ func main() {
 		setupLog.Info("Skipping setup of Knative Service validating/mutating Webhook, because KServe Serverless setup seems to be disabled in the DataScienceCluster resource.")
 	}
 
+	nimEnabled, nimEnabledErr := utils.VerifyIfComponentIsEnabled(context.Background(), mgr.GetClient(), utils.KServeWithNIMComponent)
+	if nimEnabledErr != nil {
+		setupLog.Error(kserveWithMeshEnabledErr, "could not determine if kserve has nim enabled")
+	}
+
+	if nimEnabled {
+		nimValidatorWebhookSetupErr := builder.WebhookManagedBy(mgr).
+			For(&nimv1.Account{}).
+			WithValidator(webhook.NewNimAccountValidator(mgr.GetClient())).
+			Complete()
+
+		if nimValidatorWebhookSetupErr != nil {
+			setupLog.Error(err, "unable to setup NIM validating Webhook")
+			os.Exit(1)
+		}
```

Comment:
If `nimEnabledErr != nil`, we can skip this additional check

Reply:
done